src/packages/input/AInput.vue

```diff
@@ -63,6 +64,10 @@ export default defineComponent({
       type: String,
       default: '',
     },
+    maxlength: {
+      type: String,

```

Comment:
It's better to use `Number` here and remove the default value. Vue will not bind an `undefined` value to a native HTML tag, but the empty string will.

Reply:
resolve